lib/formatter.ts

```
@@ -0,0 +1,19 @@
+import style = require('dart-style');
+
+interface FormatResult {
+  code?: string;
+  error?: string;
+}
+
+/**
+ * Execute the Dart Formatter https://github.com/dart-lang/dart_style on either
+ * a single statement or an entire Dart library.
+ * Errors are logged if code cannot be properly formatted.
```

Please fail, explosively? Silent failure is a total pain to debug later on.